modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -240,7 +240,9 @@ def _read(cls, filepath_or_buffer, **kwargs):
             # columns are on each partition.
             column_widths = None
             # Check if is list of lists
-            if isinstance(parse_dates, list) and isinstance(parse_dates[0], list):
```

Comment:
Does it make sense to just short circuit the if statement here if we don't have a valid `parse_dates`, i.e.:
```python
if parse_dates is not None:
```
becomes
```
if parse_dates is not None and parse_dates != []
```
or something like that?

Reply:
That would work for this bug, but I prefer this version, because we should be checking that every element in the outer list is a list.

Reply:
I agree with checking every element - can't we do both? Short cut the outside if and check that every element is a list in the inner one?

Reply:
I doubt that the short circuit would improve performance by any measurable amount. I'd rather keep the code simpler by not adding it.

Reply:
Wfm!